From 21f0ec391be765da589963eda7388340cf1877b4 Mon Sep 17 00:00:00 2001
From: Leonard Crestez <leonard.crestez@nxp.com>
Date: Fri, 11 May 2018 19:58:40 +0300
Subject: [PATCH 3792/5242] MLK-18294 arm64: dts: imx8qm-mek-dom0: Grant guest
 LVDS1_I2C1 pads

commit  c38c877e109913fbe72c0f64a8de9a1e1a55bb4d from
https://source.codeaurora.org/external/imx/linux-imx.git

This fixes a pinctrl error in the guest, fixing these warnings:

imx8qm-pinctrl passthrough:iomuxc: pin_config_set op failed for pin 60
imx-lpi2c 157247000.i2c: Error applying setting, reverse things back

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Reviewed-by: Peng Fan <peng.fan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-mek-dom0.dts     |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
index 0142147..f089b56 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-dom0.dts
@@ -80,6 +80,9 @@
 				SC_R_MU_2A
 				>;
 			pads = <
+				/* i2c1_lvds1 */
+				SC_P_LVDS1_I2C1_SCL
+				SC_P_LVDS1_I2C1_SDA
 				/* emmc */
 				SC_P_EMMC0_CLK
 				SC_P_EMMC0_CMD
-- 
1.7.9.5

